docs/using-the-jdbc-driver/using-plugins/UsingTheHostMonitoringPlugin.md

```diff
@@ -72,3 +72,15 @@ properties.setProperty("monitoring-socketTimeout", "10");
 > We recommend you either disable the Host Monitoring Connection Plugin or avoid using RDS Proxy endpoints when the Host Monitoring Connection Plugin is active.
 >
 > Although using RDS Proxy endpoints with the AWS Advanced JDBC Driver with Enhanced Failure Monitoring doesn't cause any critical issues, we don't recommend this approach. The main reason is that RDS Proxy transparently re-routes requests to a single database instance. RDS Proxy decides which database instance is used based on many criteria (on a per-request basis). Switching between different instances makes the Host Monitoring Connection Plugin useless in terms of instance health monitoring because the plugin will be unable to identify which instance it's connected to, and which one it's monitoring. This could result in false positive failure detections. At the same time, the plugin will still proactively monitor network connectivity to RDS Proxy endpoints and report outages back to a user application if they occur.
+
+# Host Monitoring Plugin v2
+This plugin is an alternative implementation of host health status monitoring and it's functionally equal to Host Monitoring Plugin described above. Both plugins share the same set of configuration parameters as mentioned above. `efm2` plugin could be used in any scenarios where `efm` plugin is mentioned. Since these two plugin are two different plugins, user may decide to use them together within a single connection. Such usage won't suppose to make any negative side effects. However, it's not recommended. It's recommended to use either `efm` plugin, or `efm2` plugin where it's needed. 
```

Comment:
```suggestion
This plugin is an alternative implementation of host health status monitoring and it is functionally equal to the Host Monitoring Plugin described above. Both plugins share the same set of [configuration parameters](#enhanced-failure-monitoring-parameters). The `efm2` plugin can be used in any scenario where the `efm` plugin is mentioned. Since these two plugins are separate plugins, users may decide to use them together with a single connection. While this should not  have any negative side effects, it is not recommended. It is recommended to use either the `efm` plugin, or the `efm2` plugin where it's needed. 
```